Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM fedora
-RUN dnf install -y git findutils systemd grub2-tools-minimal
+RUN dnf install -y git findutils systemd grub2-tools-minimal util-linux
 
 RUN git clone https://github.com/bats-core/bats-core.git
 WORKDIR /bats-core
 RUN ./install.sh /usr/local
 
 COPY ./usr/libexec/greenboot /usr/libexec/greenboot
-COPY ./etc/greenboot/check /etc/greenboot/check
+COPY ./usr/etc/greenboot/check /usr/etc/greenboot/check
```

Comment:
I think this should be moved to `/usr/lib/`, not `/usr/etc/`

Reply:
You're right. Updated.